lib/terraform/runner/variables_response.rb

```diff
@@ -0,0 +1,29 @@
+require 'json'
+
+module Terraform
+  class Runner
+    # Response object designed for holding full response from terraform-runner api/template/variables
+    class VariablesResponse
+      include Vmdb::Logging
+
+      attr_reader :template_input_params, :template_output_params, :terraform_version
+
+      # @return [String] Extracted attributes from the JSON response body object
+      def self.parsed_response(http_response)
+        data = JSON.parse(http_response.body)
+        _log.debug("data : #{data}")
+        Terraform::Runner::VariablesResponse.new(
+          :template_input_params  => data['template_input_params'],
+          :template_output_params => data['template_output_params'],
+          :terraform_version      => data['terraform_version']
+        )
```

Comment:
Do we need to create a new response class when we could just return the payload from the api call?  It doesn't look like there is much translation being done other than string to symbol keys :shrug: